exporter/clickhouselogsexporter/exporter.go

```diff
@@ -184,13 +185,12 @@ func (e *clickhouseLogsExporter) pushLogsData(ctx context.Context, ld plog.Logs)
 					if err != nil {
 						return err
 					}
-
 					err = statement.Append(
 						ts,
 						ots,
 						e.ksuid.String(),
-						r.TraceID().HexString(),
-						r.SpanID().HexString(),
+						utils.GetTraceId(r.TraceID()),
+						utils.GetSpanId(r.SpanID()),
```

Comment:
```suggestion
						utils.GetTraceId(r.TraceID()),
						utils.GetSpanId(r.SpanID()),
```
I would use more clear name, `ToHexString` or something.

Reply:
Done